<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-08T19:50:58 GMT-07:00" name="Default suite" finished-at="2025-07-08T19:50:58 GMT-07:00" duration-ms="53">
    <groups>
    </groups>
    <test started-at="2025-07-08T19:50:58 GMT-07:00" name="Default test" finished-at="2025-07-08T19:50:58 GMT-07:00" duration-ms="53">
      <class name="DependencyMethods.DependencyMethods">
        <test-method signature="openapp()[pri:1, instance:DependencyMethods.DependencyMethods@59fd97a8]" started-at="2025-07-08T19:50:58 GMT-07:00" name="openapp" finished-at="2025-07-08T19:50:58 GMT-07:00" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openapp -->
        <test-method signature="login()[pri:2, instance:DependencyMethods.DependencyMethods@59fd97a8]" started-at="2025-07-08T19:50:58 GMT-07:00" name="login" finished-at="2025-07-08T19:50:58 GMT-07:00" depends-on-methods="DependencyMethods.DependencyMethods.openapp" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="search()[pri:3, instance:DependencyMethods.DependencyMethods@59fd97a8]" started-at="2025-07-08T19:50:58 GMT-07:00" name="search" finished-at="2025-07-08T19:50:58 GMT-07:00" depends-on-methods="DependencyMethods.DependencyMethods.login" duration-ms="4" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at DependencyMethods.DependencyMethods.search(DependencyMethods.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- search -->
        <test-method signature="advsearch()[pri:4, instance:DependencyMethods.DependencyMethods@59fd97a8]" started-at="2025-07-08T19:50:58 GMT-07:00" name="advsearch" finished-at="2025-07-08T19:50:58 GMT-07:00" depends-on-methods="DependencyMethods.DependencyMethods.search, DependencyMethods.DependencyMethods.login" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependencyMethods.DependencyMethods.advsearch() on instance DependencyMethods.DependencyMethods@59fd97a8 depends on not successfully finished methods 
[DependencyMethods.DependencyMethods.search() on instance DependencyMethods.DependencyMethods@59fd97a8
DependencyMethods.DependencyMethods.login() on instance DependencyMethods.DependencyMethods@59fd97a8]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependencyMethods.DependencyMethods.advsearch() on instance DependencyMethods.DependencyMethods@59fd97a8 depends on not successfully finished methods 
[DependencyMethods.DependencyMethods.search() on instance DependencyMethods.DependencyMethods@59fd97a8
DependencyMethods.DependencyMethods.login() on instance DependencyMethods.DependencyMethods@59fd97a8]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- advsearch -->
        <test-method signature="logout()[pri:5, instance:DependencyMethods.DependencyMethods@59fd97a8]" started-at="2025-07-08T19:50:58 GMT-07:00" name="logout" finished-at="2025-07-08T19:50:58 GMT-07:00" depends-on-methods="DependencyMethods.DependencyMethods.login" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
      </class> <!-- DependencyMethods.DependencyMethods -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
